src/components/commons/InputRadio/index.tsx

```diff
@@ -0,0 +1,43 @@
+import { useEffect, useRef } from 'react';
+
+import classNames from 'classnames/bind';
+import { useFormContext } from 'react-hook-form';
+
+import { RecruitTypes } from '@/constants';
+
+import styles from './InputRadio.module.scss';
+
+const cx = classNames.bind(styles);
+
+type InputRadioProps = {
+  legend: string;
+  name: string;
+};
+
+const InputRadio = ({ legend, name }: InputRadioProps) => {
+  const { register } = useFormContext();
+
+  const firstInputRef = useRef<HTMLLabelElement>(null);
+
+  useEffect(() => {
+    if (firstInputRef.current) {
+      firstInputRef.current.click();
+    }
+  }, []);
+
+  return (
+    <fieldset className={cx('input-radio')}>
+      <legend className={cx('input-radio-legend')}>{legend}</legend>
+      {RecruitTypes.map((option, index) => (
+        <div className={cx('input-radio-option-container')} key={option.id}>
+          <input className={cx('radio')} type='radio' id={option.id} value={option.value} {...register(name)} />
```

Comment:
```suggestion
          <input className={cx('radio')} type='radio' id={option.id} value={option.value} {...register(name)} checked={index === 0} />
```

첫번째 라디오버튼이 처음부터 선택되도록 하려고 하는거면 이렇게 인덱스에 따라 checked나 defaultChecked 옵션만 줘도 되지 않을까요?
그러면 ref 를 사용하지 않아도 되고 useEffect 부분도 없어도 될거같습니다!

[checked와 defaultChecked 차이](https://pusha.tistory.com/entry/checked-defaultChecked)

Reply:
오 defaultChecked를 사용하면 되겠네요~ 반영하겠습니다!